stacks/dual-hive-hdfs-s3/trino.yaml

```diff
@@ -86,7 +86,7 @@ metadata:
   name: opa
 spec:
   image:
-    productVersion: 0.51.0
+    productVersion: 0.67.1
```

Comment:
I guess this got missed due to it being a much older version (assuming grep was used to find what to replace).